web_src/less/_repository.less

```diff
@@ -3066,6 +3066,26 @@ td.blob-excerpt {
   padding-left: 8px;
 }
 
+#div-file-tree {
+  position: sticky;
+  top: 47px;
+  border-top: 1px solid var(--color-secondary);
+  max-height: calc(100vh - 47px);
+  overflow-y: scroll;
+  overflow-x: scroll;
```

Comment:
Use `auto` value on these, `scroll` always shows scrollbars even when not neccessary.